src/core/config/loader.ts

```diff
@@ -108,6 +108,20 @@ async function _loadUserConfig(
       if (!compatibilityDate) {
         compatibilityDate = getConf("compatibilityDate");
       }
+
+      const alias = getConf("alias");
```

Comment:
We should perhaps move it to [resolves/paths](https://github.com/nitrojs/nitro/blob/v2/src/core/config/resolvers/paths.ts)

Reply:
Yep sure I'll move it